### ISSUE TITLE
C-g leads to "max call call size exceeded"

### ISSUE THREAD

**Issue by emacsomancer** [atlas-engineer/next#82]
C-g freezes the browser. Error is:

```
*
debugger invoked on a GLIB::G-ERROR-CONDITION in thread
#<THREAD "cl-cffi-gtk main thread" RUNNING {1006063F13}>:
  GError: Domain: "WebKitJavascriptError", Code: 699, Message: undefined: RangeError: Maximum call stack size exceeded.

```

Running with sbcl on Void Linux. Perhaps a musl libc issue?

**Comment by jmercouris** [comment]
Hi, thank you for your report!

Could you please share what URL caused this crash? Maybe there were many many links on the page or something?

**Comment by Ambrevar** [comment]
This should be fixed in Next 1.2.1.  Feel free to re-open if you still experience the issue.
